webui.py

```diff
@@ -1275,12 +1306,15 @@ def packages_rss(self):
         if self.cache_redis is None:
             content = EMPTY_RSS
         else:
-            value = self.cache_redis.get('rss~pkgs')
-            if value:
-                content = value
-            else:
-                tasks.rss_regen()
-                content = self.cache_redis.get('rss~pkgs')
+            try:
+                value = self.cache_redis.get('rss~pkgs')
+                if value:
+                    content = value
+                else:
+                    tasks.rss_regen()
```

Comment:
This might be a silly question, but is there any reason why we can't just treat a connection error the same as an expired cache entry?